devel/ci/bodhi-ci

```diff
@@ -1017,14 +1017,38 @@ class IntegrationDumpDownloadJob(BuildJob):
 
         self._app_name = app_name
         self._label = 'integration-dump-download-{}'.format(app_name)
-        filepath = os.path.join("devel", "ci", "integration", "dumps", "{}.dump".format(self._app_name))
+        self.filepath = os.path.join(
+            "devel", "ci", "integration", "dumps", "{}.dump".format(self._app_name))
         url = "https://infrastructure.fedoraproject.org/infra/db-dumps/{app}.dump.xz".format(app=self._app_name)
         self._popen_kwargs['shell'] = True
-        self._command = ["curl -o {filepath}.xz -R -z {filepath}.xz {url} && xz -d --keep --force {filepath}.xz".format(url=url, filepath=filepath)]
```

Comment:
I would recommend keeping the `-R -z filepath` switches :
- `-R` sets the remote mtime on the downloaded file
- `-z filepath` does not download the file if the remote mtime is older.
I think it's complementary to the "1-day" check you do in `run()`.
Did you remove them for a specific reason? Did they not work?

Reply:
@abompard yeah there were actually errors from curl - I think the server didn't always send the date in a format that curl wanted.

That wasn't the only reason though - I also wanted to be able to mark the state as skipped so we had a clear record of what happened.

Reply:
OK !